python/pyspark/ml/classification.py

```diff
@@ -80,6 +80,14 @@ class LogisticRegression(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredicti
     DenseVector([5.5...])
     >>> model.intercept
     -2.68...
+    >>> lrm = LogisticRegression(maxIter=5, regParam=0.01, weightCol="weight",
+    ...     family="multinomial")
+    >>> modelm = lrm.fit(df)
```

Comment:
it might be nice to fit this to an actual multiclass dataset, that way we can show that it works and produces a model with `numClasses > 2`.